cmd/authy-export/authy-export.go

```diff
@@ -27,93 +28,126 @@ type deviceRegistration struct {
 }
 
 func main() {
-	// If we don't already have a registered device, prompt the user for one
-	regr, err := loadExistingDeviceRegistration()
-	if err == nil {
-		log.Println("Found existing device registration")
-	} else if os.IsNotExist(err) {
-		log.Println("No existing device registration found, will perform registration now")
-		regr, err = newInteractiveDeviceRegistration()
+	savePtr := flag.String("save", "", "Save encrypted tokens and apps to this json file")
+	loadPtr := flag.String("load", "", "Load tokens from this json file instead of the server")
+	flag.Parse()
+
+	var resp authy.AuthenticatorResponse
```

Comment:
This `AuthenticatorResponse` struct doesn't really exist as an API response, as far as I can tell. 

It seems like you're defining a format here solely for the purposes of saving and loading, so let's be accurate about what it is and avoid exporting a new public API:

```suggestion
	var resp struct {
		Tokens authy.AuthenticatorTokensResponse `json:"tokens"`
		Apps   authy.AuthenticatorAppsResponse   `json:"apps"`
	}
```